src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -580,7 +580,13 @@ const getStyles = cacheStyles((theme: Theme) => ({
 }))
 
 export const TransactionDetailsScene = connect<StateProps, DispatchProps, OwnProps>(
+  //  @ts-expect-error
   (state, { route: { params } }) => {
+    if (Object.keys(state.core.account.currencyWallets).length === 0) {
+      // HACK: We've logged out and there are no wallets. This is a patch fix until we can address this appropriately.
+      return
+    }
+
```

Comment:
Thanks, but we don't need this with the scene rewrite (which will make it in time for the release deadline now that the blockers are solved).

Reply:
Sweet I'll remove it then